### ISSUE TITLE
<mach/host_info.h> header missing

### ISSUE THREAD

**Issue by mlfarrell** [Microsoft/WinObjC#173]
This is needed to build SVGKit-ios

**Comment by MaddTheSane** [comment]
Correction: It's needed to build CocoaLumberjack, which SVGKit relies on.

**Comment by mlfarrell** [comment]
Good call.  I actually completely removed cocoa lumberjack from svgkit and was able to bypass this problem.

**Comment by antoine-haas** [comment]
CocoaLumberjack is a logging framework, and given it's not directly related to app functionality it's not in our priorities/roadmap right now. This might get revisited in the future, but won't be implemented in the short term.
